src/vendor.js

```diff
@@ -0,0 +1,3 @@
+import 'react';
+import 'react-dom';
+import 'react-bootstrap';
```

Comment:
do we anticipate needing `react-bootstrap`?  Just wasn't sure if this was something that had been discussed with @ekrasner1 as far direction on styling and component development.  I'm am not opposed, but we should avoid installing / shipping things we aren't actually using.  

Otherwise, let's just include it at the time we end up needing it.